### DISCUSSION TITLE
Committee meeting attendees

### DISCUSSION THREAD

**Discussion by russelltrow** [Green-Software-Foundation/oc#59] [General]
At our last meeting we discussed the [OC membership criteria](https://oc.greensoftware.foundation/oc-governance.html). We agreed that we would like to open the meeting up to the leads of all active projects, not just graduated.

I believe we need to invite the following leads:

* Standards / Real Time Cloud
* Standards / Software Carbon Efficiency Rating
* Standards / SCI Guide

Are there any objections to expanding the member list? At the same time would it make sense for us to do a little internal engagement to try and attract more attendees to the call?

**Comment by russelltrow** [comment]
With the 'Community / Green Software Maturity Matrix' project launching this week as well I would like to add the Project Lead.

@tmcclell @Sealjay happy for me to add all the above to the mailing list?

**Reply by russelltrow** [reply]
I will add the additional Project Leads to the mailing list on 30th November ahead of the next meeting, if no objections are raised

**Reply by tmcclell** [reply]
No objections. :)

**Reply by Sealjay** [reply]
All good here

**Comment by russelltrow** [comment]
All new Project Leads added to the mailing list
